shared/chat/routes.native.js

```diff
@@ -38,6 +38,11 @@ const conversationRoute = new RouteDefNode({
           tags: {hideStatusBar: true},
           children: {},
         },
+        showNewTeamDialog: {
+          component: NewTeamDialog,
+          tags: {layerOnTop: true},
+          children: {},
+        },
```

Comment:
Do we still need the `showNewTeamDialog` root child below? Some other thing uses it?

Reply:
didn't want ot mess with it at this point, doesn't hurt anything being there